site/content/en/_index.html

```diff
@@ -0,0 +1,23 @@
++++
+title = "CVAT"
+linkTitle = "CVAT"
++++
+
+{{< blocks/section height="full" color="docs" >}}
+
+<section id="temporary-page" class="mx-auto text-center py-5">
+<div class="py-4">
+	<i class="fas fa-tools fa-7x"></i>
+</div>
+<div class="py-4">
+	<h1 class="text-center">This page is in development.
```

Comment:
@TOsmanov , `This page is in development`. Is it correct?

Reply:
@nmanovic 

At the moment I am not developing the main page, the main page is in site/content/en/noindex.html